eth/ethconfig/gen_config.go

```diff
@@ -95,6 +95,7 @@ func (c Config) MarshalTOML() (interface{}, error) {
 	enc.StateHistory = c.StateHistory
 	enc.StateScheme = c.StateScheme
 	enc.PathSyncFlush = c.PathSyncFlush
+	enc.JournalFileEnabled = c.JournalFileEnabled
```

Comment:
@jingjunLi
after add new field to ethconfig, gen_config.go should be regenerated